ark/utils/notebooks_test.py

```diff
@@ -123,6 +128,42 @@ def test_segment_image_data_folder(tb):
         notebooks_test_utils.create_exp_mat(tb, nuclear_counts=True)
 
 
+# # test mibitiff clustering
+# @testbook(PIXEL_CLUSTER_PATH, timeout=6000)
+# def test_pixel_clustering_mibitiff(tb):
+#     with tdir() as base_dir:
+#         # create input files
+#         notebooks_test_utils.flowsom_setup(tb, flowsom_dir=base_dir, is_mibitiff=True)
+
+#         # load img data in
+#         notebooks_test_utils.flowsom_set_fovs_channels(tb,
+#                                                        channels=['chan0', 'chan1'],
+#                                                        fovs=['fov0_otherinfo-MassCorrected-Filtered.tiff',
+#                                                              'fov1-MassCorrected-Filtered.tiff'])
+
+#         # run the FlowSOM preprocessing and clustering
+#         notebooks_test_utils.flowsom_run(tb,
+#                                          fovs=['fov0_otherinfo-MassCorrected-Filtered.tiff',
+#                                                'fov1-MassCorrected-Filtered.tiff'],
+#                                          channels=['chan0', 'chan1'],
+#                                          is_mibitiff=True)
+
+
+# test folder clustering
+@testbook(PIXEL_CLUSTER_PATH, timeout=6000)
+def test_pixel_clustering_folder(tb):
+    with tdir() as base_dir:
+        # create input files
+        notebooks_test_utils.flowsom_setup(tb, flowsom_dir=base_dir)
+
+        # run the FlowSOM preprocessing and clustering
+        notebooks_test_utils.flowsom_pixel_run(
+            tb, fovs=['fov0', 'fov1'], channels=['chan0', 'chan1']
+        )
+
+        # TODO: see what Brian discovers about R testing, then add cell clustering tests
```

Comment:
How long does the pixel clustering tests take? Can we add a simple cell clustering one to make sure the notebook runs?